.github/workflows/ci.yml

```diff
@@ -0,0 +1,83 @@
+name: 'GH Actions CI'
+
+on:
+  push:
+    branches: ['*master', '*0.[0-9].x']
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: ['*master', '*0.[0-9].x']
+  # schedule:
+  #   - cron: '0 11 * * 3'
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - FROM:     'ubuntu:focal'
+            COMPILER: 'gcc'
+            FLAGS:    '-DUSE_PYTHON_3=OFF'
+          - FROM:     'ubuntu:focal'
+            COMPILER: 'clang'
+            FLAGS:    '-DUSE_PYTHON_3=OFF'
+          - FROM:     'ubuntu:focal'
+            COMPILER: 'clang'
+            FLAGS:    '-DUSE_PYTHON_3=ON'
+          - FROM:     'ubuntu:bionic'
+            COMPILER: 'gcc'
+            FLAGS:    '-DUSE_PYTHON_3=OFF'
+          - FROM:     'ubuntu:bionic'
+            COMPILER: 'clang'
+            FLAGS:    '-DUSE_PYTHON_3=OFF'
+          - FROM:     'ubuntu:bionic'
+            COMPILER: 'clang'
+            FLAGS:    '-DUSE_PYTHON_3=ON'
+          - FROM:     'ubuntu:xenial'
+            COMPILER: 'gcc'
+            FLAGS:    '-DUSE_PYTHON_3=OFF'
+          - FROM:     'ubuntu:xenial'
+            COMPILER: 'gcc'
+            FLAGS:    '-DUSE_PYTHON_3=ON'
+          - FROM:     'debian:buster'
+            COMPILER: 'clang'
+            FLAGS:    '-DUSE_PYTHON_3=ON'
+          - FROM:     'debian:stretch'
+            COMPILER: 'gcc'
+            FLAGS:    '-DUSE_PYTHON_3=OFF'
+          - FROM:     'opensuse/leap'
+            COMPILER: 'clang'
+            FLAGS:    '-DUSE_PYTHON_3=ON'
+          - FROM:     'fedora:33'
+            COMPILER: 'clang'
+            FLAGS:    '-DUSE_PYTHON_3=ON'
+          - FROM:     'fedora:32'
+            COMPILER: 'clang'
+            FLAGS:    '-DUSE_PYTHON_3=ON'
+          - FROM:     'centos:8'
+            COMPILER: 'clang'
+            FLAGS:    '-DUSE_PYTHON_3=ON'
```

Comment:
so we lose our Mac builds?

Reply:
Yes, we do lose our Mac builds. For now.